src/param/mod.rs

```diff
@@ -27,12 +29,13 @@ use param_type::*;
 
 use bytes::{Buf, Bytes, BytesMut};
 use std::{any::Any, fmt};
+use crate::param::param_unknown::ParamUnknown;
 
 pub(crate) trait Param: fmt::Display + fmt::Debug {
     fn header(&self) -> ParamHeader;
     fn unmarshal(raw: &Bytes) -> Result<Self>
-    where
-        Self: Sized;
+        where
+            Self: Sized;
```

Comment:
Are you sure this is `cargo fmt`? This whitespace should not be added I think.

Reply:
This might have been caused by some weird interaction with my editor. I'll make sure to run cargo fmt when I push the commit for the doc comments.

Reply:
ran cargo fmt with the last commit